torch/_C/_profiler.pyi

```diff
@@ -35,6 +35,7 @@ class _ExperimentalConfig:
         self,
         profiler_metrics: List[str] = ...,
         profiler_measure_per_kernel: bool = ...,
+        no_callstack_trace: bool = ...,
```

Comment:
I really like the idea of putting this flag in ExperimentalConfig. I think we should make it more generic. Maybe `new_trace_format`? (@aaronenyeshi, @chaekit feel free to bikeshed.)

Reply:
will mark resolved unless we are trying `new_trace_format` in this PR.

Reply:
I was mostly thinking just because that way we wouldn't have to update the name every time we want to improve the trace format. The code itself doesn't have to change. (Maybe `updated_format`?)

Reply:
Good point. I chose `verbose` (more general) for this PR changes instead of `no_callstack_trace`, with an opposite default value and comment. I guess we would update trace format in the future but not only just reducing it but improve in a different way?

Reply:
Ohh, I like `verbose`.